com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -509,6 +529,11 @@ private void SendSceneEventData(uint sceneEventId, ulong[] targetClientIds)
         /// <returns>true (Valid) or false (Invalid)</returns>
         internal bool IsSceneNameValid(string sceneName)
         {
+            if (sceneName.Contains("/"))
```

Comment:
Yeah, again I start getting the jitters thinking this must be something we can more directly in the engine.  Then again I have not dug into it.   

Lastly, I assume this works on all platforms (e.g. \ vs. / )

Reply:
Yeah, the one to watch out for is the "\" as that typically requires "\\".  "/" is the better/more universal way to handle paths.